index.bs

```diff
@@ -7945,6 +7966,38 @@ This section defines a number of
 whose presence affects only the ECMAScript binding.
 
 
+<h4 id="AllowShared" extended-attribute lt="AllowShared">[AllowShared]</h4>
+
+If the [{{AllowShared}}] [=extended attribute=] appears on one of the [=buffer source types=], it
+creates a new IDL type that allows the buffer source type to be backed by an ECMAScript
+{{ECMAScript/SharedArrayBuffer}}, instead of only by a non-shared {{ECMAScript/ArrayBuffer}}.
+
+The [{{AllowShared}}] extended attribute must [=takes no arguments|take no arguments=].
+
+A type that is not a [=buffer source type=] must not be
+[=extended attributes associated with|associated with=] the [{{AllowShared}}] extended attribute.
+
+See the rules for converting ECMAScript values to IDL [=buffer source types=] in
```

Comment:
This is already used for every other extended attribute, so I'd rather be consistent.